packages/relay/src/lib/errors.ts

```diff
@@ -70,5 +70,10 @@ export const predefined = {
     name: 'Incorrect nonce',
     code: -32006,
     message: 'Incorrect nonce'
+  }),
+  'UNSUPPORTED_CHAIN_ID': new JsonRpcError({
+    name: 'ChainId not supported',
+    code: -32000,
+    message: 'ChainId not supported'
```

Comment:
Q: do you think we should note the incorrect and correct chainId in the message.
Non blocking comment we can do later if it makes sense.
For now so far as we get some documentation with the right chainId on the repo it should be fine

Reply:
Can you add this simple table below [here](https://github.com/hashgraph/hedera-json-rpc-relay#configuration-1). This will provide the reference I noted above
```
The following table highlights some initial configuration values to consider
| Config      | Default | Description           |
| ------------|---------|-----------------------|
| `CHAIN_ID`  | '0x12a' | The netowrk chain id. Local and previewnet envs should use `0x12a`. Mainnet and Testnet should use 0x127 and `0x128` respectively |
```

Reply:
Yes, that makes sense. It would be really helpful to anyone trying to use the relay to see what the correct chainId is in the error message.